packages/block-library/src/group/edit.native.js

```diff
@@ -1,13 +1,17 @@
 /**
  * External dependencies
  */
-import { View } from 'react-native';
+import { View, Dimensions } from 'react-native';
```

Comment:
Do we need changes in the `group/edit.native.js` in this PR?

Reply:
IIRC we need that changes to properly style appender in full width group which is nested in the column. Also we are passing `parentWidth` to nested block within `Group`